storybook/.storybook/main.js

```diff
@@ -29,7 +34,11 @@ module.exports = {
       ],
     },
     resolve: {
-      ...storybookConfig.resolve,
+      ...webpackConfig.resolve,
+      modules: [
+        ...webpackConfig.resolve.modules,
```

Comment:
adding `storybook/node_modules` to the list of modules where webpack will look for modules